src/components/Link.utils.tsx

```diff
@@ -0,0 +1,36 @@
+import { Icon } from 'design-system-react';
+import type { ReactElement } from 'react';
+
+/**
+ * Programmatically determine if a link is external to the CFPB sphere of websites
+ */
+export const isExternalLink = (targetUrl: string): boolean => {
+  let parsed;
+
+  try {
+    parsed = new URL(targetUrl);
+  } catch {
+    return false; // Internal targets will fail parsing (ex. '/home')
+  }
+
+  const externalProtocols = ['http', 'mailto:', 'tel:', 'sms:', 'ftp:'];
+  if (externalProtocols.includes(parsed.protocol)) return true;
+
+  // Any subdomain of consumerfinance.gov or the current host
+  const isInternal = new RegExp(
+    `([\\S]*\\.)?(((consumerfinance|cf)\\.gov)|(${window.location.host}))`,
```

Comment:
Should any domains other than these be considered "internal" to the CFPB sphere of sites?
- consumerfinance.gov
- cf.gov

Reply:
That sounds right @billhimmelsbach

Reply:
@meissadia 

Can we make the following links _internal_: 
- links to consumerfinance.gov pages
- links within the platform
- links to cfpb.gov email addresses (that open up an email client)

Reply:
Thank you both! PR updated based on feedback.